mlir/include/mlir/Dialect/EmitC/IR/EmitC.td

```diff
@@ -56,6 +57,51 @@ def IntegerIndexOrOpaqueType : Type<CPred<"emitc::isIntegerIndexOrOpaqueType($_s
 "integer, index or opaque type supported by EmitC">;
 def FloatIntegerIndexOrOpaqueType : AnyTypeOf<[EmitCFloatType, IntegerIndexOrOpaqueType]>;
 
+def EmitC_FileOp
+    : EmitC_Op<"file", [IsolatedFromAbove, NoRegionArguments, SymbolTable,
+                        OpAsmOpInterface]#GraphRegionNoTerminator.traits> {
+  let summary = "A file container operation";
+  let description = [{
+    A `file` represents a single C/C++ file.
+
+    `mlir-translate` emits only the file selected via
+    the `-translation-unit-id=id` flag. By default, no file is emitted.
```

Comment:
Should be
```suggestion
    the `-file-id=id` flag. By default, no file is emitted.
```
or
```suggestion
    the `--file-id=id` flag. By default, no file is emitted.
```
instead. Furthermore, to clarify "By default, no file is emitted." sounds like the default behavior changes and that nothing is emitted when you don't have an `emitc.file` at all in your MLIR file.

Reply:
Updated in c7261f689d7f, does that work?

Reply:
I think it does but to clarify (sorry, traveling and not much time to get into the code), if there is an `emitc.file` and one doesn't specify `--file-id`, what is in the `emitc.file` will be "ignored" and not emitted?

Reply:
If there is a `emitc.file` and one doesn't specify `--file-id`, then the body of that `emitc.file` will NOT be emitted.